alws/crud.py

```diff
@@ -370,6 +430,63 @@ async def build_done(
         await db.commit()
 
 
+async def sign_done(
```

Comment:
Same logic as in `albs-web-server.alws.crud.build_done`. Does that really necessary?